.github/workflows/main.yml

```diff
@@ -47,64 +48,79 @@ jobs:
       - name: Prepare
         id: prepare
         run: |
-          echo 'Porcessing git ref:' $GITHUB_REF
-          
+          echo 'Processing git ref:' $GITHUB_REF
+
           DOCKER_IMAGE=isoblue/${{ matrix.service }}
 
-          if [[ $GITHUB_REF == refs/tags/* ]]; then
-            VERSION=${GITHUB_REF#refs/tags/}
-          
+          # Release version is just the release number
+          if [[ $GITHUB_REF == refs/heads/release/* ]]; then
+            VERSION=${GITHUB_REF#refs/heads/release/}
```

Comment:
One future improvement could be to check for illegal chars here (slashes) or otherwise enforce the semantic versioning style. IE if it does not match `^[0-9]\.[0-9]\.[0-9]$` exit with an error